tests/ut_main.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import database
+import EScavange
+import gamemaker
+
+class TestDatabaseCreation(unittest.TestCase):
+  #Test that we can access the database. Should not test that the database DS is correct, that should be
+  #covered by the database tests.
+  
+  def setUp(self):
+    self.db = database.Database()
+    self.gameMaker = gamemaker.GameMaker()
+  
+  def accessDB(self):
+    pass
+    #stub for now : check that we can access the database
```

Comment:
Just test a generic function. Maybe even just verify that self.db != null??

Reply:
I'll check that database != null.